src/rustdoc/membership.md

```diff
@@ -0,0 +1,116 @@
+  to mailing lists work. It's a very low-volume mailing list (maybe a few emails per year), it's a
+  way to communicate things to all contributors. We will not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+These groups are NOT full-fledge teams, and as such, to be part of any of these groups, you need to
+be a member of the rustdoc team.
+
+For now, only the front-end group has an official status in the
+[team repository](https://github.com/rust-lang/team) and is called `rustdoc-frontend`. If a rustdoc
+team member is interested to be part of this group, they can ask to be added into it.
+
+Being part of the front-end group doesn't change your rustdoc team membership. However
+you will be assigned for reviews on front-end pull requests and on front-end FCPs.
+
+## How promotion decisions are made
+[hdam]: #how-promotion-decisions-are-made
+
+After an individual has been contributing to rustdoc for a while, they may be nominated in the
+private Zulip rustdoc team channel by an existing team member.
+
+The rustdoc team members will check to see if there are concerns with extending a membership
+invitation to the individual and after a week (barring no objections), an invitation will be
+extended.
+
+If the invitation is accepted by the individual, the rustdoc team leads will update the [team]
+repository to reflect their new role.
+
+## Alumni status
+If at any time a rustdoc team member wishes to take a break from participating, they can opt to put
+themselves into alumni status. When in alumni status, they will be removed from
+GitHub aliases and the like, so that they need not be bothered with pings and messages. They will
+also not have r+ privileges. **Alumni members will however still remain members of the GitHub
+org overall.**
+
+People in alumni status can ask to return to "active" status at any time. This request would
+ordinarily be granted automatically barring extraordinary circumstances.
+
+People in alumni status are still members of the team at the level they previously attained and
+they may publicly indicate that, though they should indicate the time period for which they were
+active as well.
+
+### Automatic alumni status after 6 months of inactivity
+If a member or maintainer has been inactive in rustdoc for 6 months, then we will ask them if
+they would like to go to alumni status. If they respond yes or do not respond, they can be placed on
+alumni status. If they would prefer to remain active, that is also fine, but they will get asked
+again periodically if they continue to be inactive.
```

Comment:
It seems to me that ideally we'd have some kind of policy where people get put on alumni involuntarily (after a longer period of time, perhaps) if they aren't contributing at all, even if they say they want to stay on the team. Of course, if they explain their circumstance then there could be exceptions. And also given how alumni works, they could always be brought back to the team if/when they can contribute again.

Reply:
I think first establish such a policy, then amend the documentation.

If it helps doing things consistently across teams, T-compiler [policy](https://forge.rust-lang.org/compiler/membership.html#automatic-alumni-status-after-6-months-of-inactivity) is the same: *after X time you might be asked if you want/can continue to contribute*. Then I imagine we could open a patch on the [team repo](https://github.com/rust-lang/team) with some wording [like this](https://github.com/rust-lang/rust/pull/140579) and give the inactive contributor a chance to :+1: / :-1:

Reply:
> I think first establish such a policy, then amend the documentation.

I mean T-rustdoc doesn't have any policies at this point in time and this is the first ever document to formulate and thus *make* them.

Reply:
Absolutely. And your input is very welcome as you have more experience with compiler team policy. ;)

Reply:
This has been implemented. Marking as resolved.